packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -0,0 +1,58 @@
+name: {{project_name}}
+description: A new Flutter project.
+homepage: https://{{organization_name}}.{{domain_name}}
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  auto_route: ^1.0.2
+  cupertino_icons: ^1.0.2{{#analytics}}
+  firebase_analytics: ^8.0.1
+  firebase_core: ^1.0.4{{/analytics}}
+  flutter:
+    sdk: flutter
+  flutter_localizations:
+    sdk: flutter
+  flutter_rx_bloc: ^2.0.0
+  intl: ^0.17.0
+  json_annotation: ^4.0.1
```

Comment:
@DDavidPrime, as far as I'm aware we don't use the following packages:
* json_annotation
* retrofit

We can bring them back (conditionally) along with dio, once we start working on this ticket 
https://github.com/Prime-Holding/rx_bloc/issues/121